### PR TITLE
Added two new how to pages, fixed a few type os, and updated the ``git bash`` bug

### PR DESCRIPTION
**Pull request by lunamidori5**

### CODE REVIEW

File: content/howtos/easy-setup-full.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is for `Linux`, `Mac OS`, or `Windows` Hosts. - [Docker Desktop](https://do
 
 Linux Hosts:
 
-There is a Full_Auto installer for some types of Linuxs, feel free to use them but please note, they may not fully work so please use the links at the top if you need to install something.
+There is a Full_Auto installer for some types of Linuxes, feel free to use them, but note that they may not fully work. If you need to install something, please use the links at the top.
```
Comment (mudler) [inline content/howtos/easy-setup-full.md]:
```suggestion
There is a Full_Auto installer compatible with some types of Linux distributions, feel free to use them, but note that they may not fully work. If you need to install something, please use the links at the top.
```

File: content/howtos/_index.md [hunk 2]
```diff
@@ -14,3 +14,6 @@ This section includes LocalAI end-to-end examples, tutorial and how-tos curated
 - [Model Setup by Downloading Model]({{%relref "howtos/easy-model-import-downloaded" %}})
 - [Making requests via OpenAi API]({{%relref "howtos/easy-request-openai" %}})
 - [Making requests via Curl]({{%relref "howtos/easy-request-curl" %}})
+
+This section includes Full auto setup, install, and use of LocalAI by [lunamidori5](https://github.com/lunamidori5).
```
Comment (mudler) [inline content/howtos/_index.md:18]: this sounds a slightly repetition of https://github.com/go-skynet/localai-website/blob/c34e3fb1816fa7f8fa9058e72c9f78aaa88e7451/content/howtos/_index.md?plain=1#L9

Comment (lunamidori5) [reply content/howtos/_index.md:18]: Its the ground work for a new layout - Needs to be for now

Comment (lunamidori5) [reply content/howtos/_index.md:18]: I can remove it if you like @mudler

**Review by mudler — APPROVED** [review]
looks good!

**Pull request merged**